src/Instantiator.php

```diff
@@ -150,8 +151,11 @@ private static function accessibleProperty(object $object, string $name): \Refle
         $class = new \ReflectionClass($object);
 
         // try fetching first by exact name, if not found, try camel-case
-        if (!$property = self::reflectionProperty($class, $name)) {
-            $property = self::reflectionProperty($class, self::camel($name));
+        $property = self::reflectionProperty($class, $name);
+
+
+        if (!$property && $property = self::reflectionProperty($class, self::camel($name))) {
+            trigger_deprecation('zenstruck\foundry', '1.5.0', 'Using snake/kebab-case attributes is deprecated, use the same case as the object property instead.');
```

Comment:
This message is slightly incorrect, as it assumes the property name is camelCase. I would remove explicit case names, e.g: "Using a different case attribute is deprecated, [...]."

Reply:
Changed to: "'Using a differently cased attribute is deprecated, use the same case as the object property instead.'"